x-pack/plugins/alerting/common/rule_task_instance.ts

```diff
@@ -18,6 +18,7 @@ export const ruleStateSchema = t.partial({
 
 const ruleExecutionMetricsSchema = t.partial({
   numSearches: t.number,
+  numSearchSourceSearches: t.number,
```

Comment:
@mikecote I suggested splitting out into a separate `numSearchSourceSearches` because I thought there might be utility in keeping track of this separately. Fine with consolidating though if that makes more sense!

Reply:
Ahh I see, I think it's ok to keep it simple at first (re-use `numSearches`) and we can split if we see the need. I'm thinking it'll keep it simple for telemetry collection, rule run history UI, etc for now.